### DISCUSSION TITLE
OCI support for Helm controller

### DISCUSSION THREAD

**Discussion by hendrikhalkow** [fluxcd/flux2#1243] [Proposals]
Since [Helm is moving towards OCI](https://helm.sh/docs/topics/registries/), the Helm controller should be able to deploy Helm charts from an OCI registry, too. Here is a rough idea how this could work:

- Introduce a custom resource `Image`
- Allow the sourceRef of a `HelmRelease` to be an `Image`
- As the `Image` has a name or a tag property, it can even be auto-updated via `ImageUpdateAutomation` as described [here](https://toolkit.fluxcd.io/guides/image-update/)

**Comment by hiddeco** [comment]
Duplicate of https://github.com/fluxcd/source-controller/issues/124
